op-devstack/dsl/l2_el.go

```diff
@@ -215,3 +223,19 @@ func (el *L2ELNode) PayloadByNumber(number uint64) *eth.ExecutionPayloadEnvelope
 	el.require.NoError(err, "failed to get payload")
 	return payload
 }
+
+func (el *L2ELNode) NewPayload(ref *L2ELNode, number uint64) *NewPayloadResult {
```

Comment:
I am not sure why we need this method. Why would we get the `PayloadByNumber` from the EL, only to give it back to the EL with `NewPayload`?